feature-pagination/src/main/java/net/kyori/text/feature/pagination/Pagination.java

```diff
@@ -202,6 +203,14 @@ public String toString() {
         .build();
     }
 
+    default @NonNull Component renderLine(final char character, final @NonNull Style style, final int length) {
+      return TextComponent.of(repeat(String.valueOf(character), length), style);
+    }
+
+    static @NonNull String repeat(final @NonNull String character, final int count) {
+      return String.join("", Collections.nCopies(count, character));
```

Comment:
I'd prefer a for loop and stringbuilder instead of a short line of code that adds extra overhead

Reply:
Should be resolved now with latest commit